docs/index.rst

```diff
@@ -6,45 +6,12 @@
 PlasmaPy Documentation
 ######################
 
-.. raw:: html
-
-  <div class="sphx-glr-thumbcontainer">
-    <div class="figure align-center">
-      <img alt="thumbnail" src="./_images/notebooks_physics_16_0.png" />
-      <p class="caption">
-        <span class="caption-text">
-          <a class="reference internal" href="notebooks/physics.html">
-            <span class="std std-ref">Analysing ITER parameters</span>
-          </a>
-        </span>
-      </p>
-    </div>
-  </div>
-  <div class="sphx-glr-thumbcontainer">
-    <div class="figure align-center">
-      <img alt="thumbnail" src="./_images/notebooks_dispersion_function_9_0.png" />
-      <p class="caption">
-        <span class="caption-text">
-          <a class="reference internal" href="notebooks/dispersion_function.html">
-            <span class="std std-ref">The plasma dispersion function</span>
-          </a>
-        </span>
-      </p>
-    </div>
-  </div>
-  <div class="sphx-glr-thumbcontainer">
-    <div class="figure align-center">
-      <img alt="thumbnail" src="./_images/notebooks_langmuir_analysis_6_1.png" />
-      <p class="caption">
-        <span class="caption-text">
-          <a class="reference internal" href="notebooks/langmuir_analysis.html">
-            <span class="std std-ref">Langmuir probe data analysis</span>
-          </a>
-        </span>
-      </p>
-    </div>
-  </div>
-  <div class="sphx-glr-clear"></div>
+.. nbgallery::
+   :hidden:
+
+   notebooks/physics
+   notebooks/dispersion_function
+   notebooks/thomson
```

Comment:
[I'll let Gimli, son of Gloin, speak for me here.](https://www.youtube.com/watch?v=7qF13urEo_Y)